buddyboss/buddyboss-platform
# Validation is showing before successful registration.

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #569.

### How to test the changes in this Pull Request:

1. Go to '/register'.
2. Enter fresh details.
3. The notices appear.

### Proof Screenshots or Video
http://somup.com/cYn2oqhDQz


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
